app/src/main/kotlin/co/touchlab/droidconandroid/EventDetailFragment.kt

```diff
@@ -99,31 +91,25 @@ class EventDetailFragment() : Fragment(), EventDetailHost
      * Gets the track ID argument. This is to make sure we don't flash the incorrect colors
      * on things like the FAB and toolbar while waiting to load the event details
      */
-    private fun findTrackIdArg(): String?
-    {
+    private fun findTrackIdArg(): String? {
         var trackId = arguments?.getString(TRACK_ID)
-        if (trackId == null)
-        {
+        if (trackId == null) {
             trackId = activity.intent.getStringExtra(TRACK_ID)
         }
 
         return trackId
     }
 
-    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View?
-    {
-        presenter = EventDetailPresenter(context, findEventIdArg(), this)
-
-        return inflater !!.inflate(R.layout.fragment_event_detail, null)
+    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
I *think* that it will let you take the '?'s off of the inflater parameter and just treat it as non-null